prow/tide/tide_test.go

```diff
@@ -1233,7 +1248,7 @@ func TestHeadContexts(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Logf("Running test case %q", tc.name)
-		fgc := &fgc{}
+		fgc := &fgc{combinedStatus: map[string]string{win: "SUCCESS"}}
```

Comment:
Do we want to use the GithubQL types instead of raw string literals for the context status?

Reply:
That would be changing the signature of the interface method.

Reply:
but I used the githubql const.